### PR TITLE
sync operator prs.

### PR DESCRIPTION
**Pull request by morvencao**
Sync operator PRs:

https://github.com/istio/operator/pull/749
https://github.com/istio/operator/pull/750
https://github.com/istio/operator/pull/751

[ ] Configuration Infrastructure
[ ] Docs
[ X ] Installation
[ ] Networking
[ ] Performance and Scalability
[ ] Policies and Telemetry
[ ] Security
[ ] Test and Release
[ ] User Experience
[ ] Developer Infrastructure

### CODE REVIEW

File: operator/cmd/mesh/testdata/operator/output/operator-init.yaml [hunk 1]
```diff
@@ -162,11 +162,11 @@ spec:
         status:
           description: 'Status describes each of istio control plane component status at the current time.
             0 means NONE, 1 means UPDATING, 2 means HEALTHY, 3 means ERROR, 4 means RECONCILING.
-            More info: https://github.com/istio/operator/blob/master/pkg/apis/istio/v1alpha2/v1alpha2.pb.html &
+            More info: https://github.com/istio/api/blob/master/operator/v1alpha1/istio.operator.v1alpha1.pb.html &
```
Comment (ostromart) [inline operator/cmd/mesh/testdata/operator/output/operator-init.yaml]: This should actually now point to istio/api/operator

Comment (morvencao) [reply operator/cmd/mesh/testdata/operator/output/operator-init.yaml]: Yes, it is https://github.com/istio/api/blob/master/operator/v1alpha1/istio.operator.v1alpha1.pb.html

**Comment by morvencao** [comment]
@ostromart I found https://github.com/istio/istio/pull/20176 is out of date, can you rebase or merge this one instead?

**Comment by howardjohn** [comment]
Assuming this has been synced by now..

**Comment by sdake** [comment]
https://github.com/istio/istio/pull/20176 was closed and this was closed.

**Comment by morvencao** [comment]
Closing as all PRs have been synced.
/close

**Pull request closed without merging**